### DISCUSSION TITLE
Dutch (Nederlands)

### DISCUSSION THREAD

**Discussion by Binsky21** [sovenor/bitcoin-rocks#51] [Translations]
I'm currently working on the Dutch translation of this website. All this is fairly new to me, so I hope it goes well. wish me luck

**Comment by ObjectifMoon21** [comment]
Hi Binsky ! You can do it 💪 Good Luck !

**Comment by sovenor** [comment]
Good luck, and thank you for getting involved! Let me know if you have any questions.

**Comment by sovenor** [comment]
Hi @Binsky21 are you still working on this translation?

**Reply by Binsky21** [reply]
Hi there,

still working on it indeed. It's taking me a bit longer than expected tbh, but I'm getting there. I could upload some files already if you'd like that?

**Reply by Binsky21** [reply]
@sovenor 

i can still make some adjustments to the Dutch translation, I noticed some mistakes

**Reply by sovenor** [reply]
No worries, thanks for your continued work! Feel free to open a pull request whenever you're ready to share your work. Let me know if you need any help doing that. @Binsky21
